services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/internal/operations/UpdateItemOperation.java

```diff
@@ -145,6 +151,56 @@ public UpdateItemRequest generateRequest(TableSchema<T> tableSchema,
 
         return requestBuilder.build();
     }
+    
+    /**
+     * Method checks if a nested object parameter requires an update
+     * If so flattens out nested params separated by "_NESTED_ATTR_UPDATE_"
+     * this is consumed by @link EnhancedClientUtils to form the appropriate UpdateExpression
+     */
+    public Map<String, AttributeValue> transformItemToMapForUpdateExpression(Map<String, AttributeValue> itemToMap,
+                                                                             boolean ignoreNulls) {
```

Comment:
Can we simplify this method since as far as I can see, `ignoreNulls` is always true? Can we just remove it - this is an internal class.

Reply:
Good catch, i can do that

Reply:
fixed